pipelines/rj_crm__disparo_template/flow.py

```
@@ -189,173 +196,194 @@ def rj_crm__disparo_template(
+        if i == 0:
+            current_df = base_df.copy()
+            # No primeiro disparo, exclui linhas com telefone None (marcadas por filter_failed_phones)
+            current_df = current_df.dropna(subset=["telefone"])
+        else:
+            if "others" in base_df.columns and not base_df["others"].apply(
+                lambda x: isinstance(x, list) and len(x) >= i
+            ).any():
+                print(f"✅ No others available for retry attempt {i}. Ending retry loop.")
+                break
+
+            print(f"⚠️  Sleep 3 minutes before retry dispatch.")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)